libs/api/books/src/lib/reading-list.controller.ts

```diff
@@ -20,4 +20,9 @@ export class ReadingListController {
   async removeFromReadingList(@Param() params) {
     return await this.readingList.removeBook(params.id);
   }
+
+  @Put('/reading-list/:id/finished')
```

Comment:
why the task 4 changes included in task 1? please include only code review changes and improvements